Apromore-Core-Components/Apromore-Portal/src/main/java/org/apromore/portal/common/access/Assignment.java

```diff
@@ -0,0 +1,72 @@
+/*-
+ * #%L
+ * This file is part of "Apromore Core".
+ *
+ * Copyright (C) 2011 Marie Christine.
+ * Copyright (C) 2012 - 2017 Queensland University of Technology.
+ * %%
+ * Copyright (C) 2018 - 2020 Apromore Pty Ltd.
+ * %%
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Lesser Public License for more details.
+ *
+ * You should have received a copy of the GNU General Lesser Public
+ * License along with this program.  If not, see
+ * <http://www.gnu.org/licenses/lgpl-3.0.html>.
+ * #L%
+ */
+
+package org.apromore.portal.common.access;
+
+import org.apromore.dao.model.Group.Type;
+import org.apromore.util.AccessType;
+
+/**
+ * Assignment model for listmodel-based UI
+ */
+public class Assignment {
+    public String name;
+    public String rowGuid;
+    public Type type;
+    public boolean isGroup;
+    public String access; // Viewer, Editor, Owner (UI front for AccessType)
```

Comment:
Use the existing Enum here. U can use Capitalize to get the Capital String.
Also U can get the Enum back from the String using AccessType.valueOf(String) instead of the method.